src/Form/ServerForm.php

```diff
@@ -89,7 +89,15 @@ public function form(array $form, FormStateInterface $formState) {
     $schemas = array_map(function ($definition) {
       return $definition['name'] ?? $definition['id'];
     }, $this->schemaManager->getDefinitions());
-    $schema = ($formState->getUserInput()['schema'] ?: $server->get('schema')) ?: reset(array_keys($schemas));
```

Comment:
I think this is not the correct fix. I think the author wanted to use the `??` operator instead of `?:` here. That should be equivalent to what you are trying to do.